backend/Wheelingful.Services/Outer/BookAuthorService.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Wheelingful.Core.Contracts.Auth;
+using Wheelingful.Core.Contracts.Books;
+using Wheelingful.Core.DTO.Books;
+using Wheelingful.Services.Extensions;
+
+namespace Wheelingful.Services.Outer;
+
+internal class BookAuthorService : IBookAuthorService
+{
+    private readonly IBookCoverManager _bookCover;
+    private readonly IBookManager _bookManager;
+    private readonly ICurrentUser _currentUser;
+
+    public BookAuthorService(IBookManager bookManager, IBookCoverManager bookCover, ICurrentUser currentUser)
+    {
+        _bookCover = bookCover;
+        _bookManager = bookManager;
+        _currentUser = currentUser;
+    }
+
+    public async Task CreateBook(NewBookModel model)
+    {
+        model.AuthorId = _currentUser.Id;
+
+        model.CoverId = await _bookCover.UploadCover(model.CoverBase64, model.Title, model.AuthorId);
+
+        await _bookManager.CreateBook(model);
+    }
+
+    public async Task UpdateBook(UpdatedBookModel model)
+    {
+        var oldCoverTask = _bookManager
+            .GetBooks()
+            .Where(b => b.Id == model.Id)
+            .Select(b => b.CoverId)
+            .FirstAsync();
+
+        var check = await _bookManager.IsActualAuthor(model.Id, _currentUser.Id);
+
+        if (check is false)
+        {
+            return;
+        }
+
+        if (model.CoverBase64 != null)
```

Comment:
somewhere you use "is" somewhere you use "==". consider using one style

Reply:
The "is" you have seen before was in the deleted authentication solution. But I will consider it anyway.

Reply:
if (check is false) vs if (!check)

Reply:
also think about the name of the variable.